packages/contracts/deploy/src/utils/network.ts

```diff
@@ -1,15 +1,14 @@
-import promisify = require('es6-promisify');
+import {promisify} from '@0xproject/utils';
+import {Web3Wrapper} from '@0xproject/web3-wrapper';
```

Comment:
Wait, first of all, we aren't using this here. Second, why do we need this module at all? Can't we just call `web3Wrapper`'s `getNetworkIdAsync` instead?

Reply:
current web3Wrapper assumes that networkId is already passed from outside.

Reply:
Maybe `networkId` should be optionally passed in. It isn't actually used in the constructor... And the method for getting the networkId from web3 should still be there.

Reply:
Happy to discuss this over Appear.in.